src/liberty/devCommands.ts

```diff
@@ -77,18 +77,17 @@ export async function openProject(pomPath: string): Promise<void> {
 export async function listAllCommands(): Promise<void> {
     const libertyCommands = Array.from(COMMAND_TITLES.keys());
     vscode.window.showQuickPick(libertyCommands).then(selection => {
-            if (!selection) {
-                return;
-            }
-            const command = COMMAND_TITLES.get(selection);
-            if ( command !== undefined )
-            {
-                vscode.commands.executeCommand(command);
-            } else {
-                // should never happen
-                console.error("Unable to find corresponding command for " + selection);
-            }
-                
+        if (!selection) {
+            return;
+        }
+        const command = COMMAND_TITLES.get(selection);
+        if (command !== undefined) {
+            vscode.commands.executeCommand(command);
+        } else {
+            // should never happen
+            console.error("Unable to find corresponding command for " + selection);
+        }
+
```

Comment:
Please do not do any formatting changes to code that is not added or modified by you for this PR. It makes the changes in the PR more difficult to follow. To fix formatting in existing code, we should open separate PRs that are specifically for fixing formatting.

For instance, reviewing this section, it is not immediately clear to me if there was a functional change here or not.

Reply:
Understood , removed the formatting which is not part of this PR